### ISSUE TITLE
Lora精调时发现loss=0.0

### ISSUE THREAD

**Issue by zhangzhenyu13** [LianjiaTech/BELLE#198]
Bloom-7B的模型
deepspeed运行8卡v100 16g
loss=0

You're using a BloomTokenizerFast tokenizer. Please note that with a fast tokenizer, using the `__call__` method is faster than using a method to encode the text followed by a call to the `pad` method to get a padded encoding.
You're using a BloomTokenizerFast tokenizer. Please note that with a fast tokenizer, using the `__call__` method is faster than using a method to encode the text followed by a call to the `pad` method to get a padded encoding.
You're using a BloomTokenizerFast tokenizer. Please note that with a fast tokenizer, using the `__call__` method is faster than using a method to encode the text followed by a call to the `pad` method to get a padded encoding.
{'loss': 0.0, 'learning_rate': 7.8125e-07, 'epoch': 0.01}                                                                                                                                                                                          
{'loss': 0.0, 'learning_rate': 1.5625e-06, 'epoch': 0.02}                                                                                                                                                                                          
{'loss': 0.0, 'learning_rate': 2.3437500000000002e-06, 'epoch': 0.02}                                                                                                                                                                              
{'loss': 0.0, 'learning_rate': 3.125e-06, 'epoch': 0.03}                                                                                                                                                                                           
{'loss': 0.0, 'learning_rate': 3.90625e-06, 'epoch': 0.04}                                                                                                                                                                                         
{'loss': 0.0, 'learning_rate': 4.6875000000000004e-06, 'epoch': 0.05}                                                                                                                                                                              
{'loss': 0.0, 'learning_rate': 5.468750000000001e-06, 'epoch': 0.05}                                                                                                                                                                               
{'loss': 0.0, 'learning_rate': 6.25e-06, 'epoch': 0.06}                                                                                                                                                                                            
{'loss': 0.0, 'learning_rate': 7.031250000000001e-06, 'epoch': 0.07}                                                                                                                                                                               
{'loss': 0.0, 'learning_rate': 7.8125e-06, 'epoch': 0.08}

**Comment by zhangzhenyu13** [comment]
重启多次都是如此

**Comment by hangzeli08** [comment]
我也是

**Comment by xianghuisun** [comment]
> Bloom-7B的模型 deepspeed运行8卡v100 16g loss=0
> 
> You're using a BloomTokenizerFast tokenizer. Please note that with a fast tokenizer, using the `__call__` method is faster than using a method to encode the text followed by a call to the `pad` method to get a padded encoding. You're using a BloomTokenizerFast tokenizer. Please note that with a fast tokenizer, using the `__call__` method is faster than using a method to encode the text followed by a call to the `pad` method to get a padded encoding. You're using a BloomTokenizerFast tokenizer. Please note that with a fast tokenizer, using the `__call__` method is faster than using a method to encode the text followed by a call to the `pad` method to get a padded encoding. {'loss': 0.0, 'learning_rate': 7.8125e-07, 'epoch': 0.01} {'loss': 0.0, 'learning_rate': 1.5625e-06, 'epoch': 0.02} {'loss': 0.0, 'learning_rate': 2.3437500000000002e-06, 'epoch': 0.02} {'loss': 0.0, 'learning_rate': 3.125e-06, 'epoch': 0.03} {'loss': 0.0, 'learning_rate': 3.90625e-06, 'epoch': 0.04} {'loss': 0.0, 'learning_rate': 4.6875000000000004e-06, 'epoch': 0.05} {'loss': 0.0, 'learning_rate': 5.468750000000001e-06, 'epoch': 0.05} {'loss': 0.0, 'learning_rate': 6.25e-06, 'epoch': 0.06} {'loss': 0.0, 'learning_rate': 7.031250000000001e-06, 'epoch': 0.07} {'loss': 0.0, 'learning_rate': 7.8125e-06, 'epoch': 0.08}

目前在V100上训练Bloom模型是会存在这个问题
可参考：https://github.com/LianjiaTech/BELLE/issues/122
